pgmpy/factors/hybrid/FunctionalCPD.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+from pgmpy.factors.base import BaseFactor
+
+
+class FunctionalCPD(BaseFactor):
+    def __init__(self, variable, fn, parents=None):
+        """
+        Parameters:
+        ----------
+        variable: str
+            Name of the variable for which this CPD is defined.
+        fn: callable
+            A function that takes a dictionary of parent variable values
+            and returns a sampled value for the variable.
+        parents: list[str], optional
+            List of parent variable names (default is None for no parents).
+
+        Examples
+        --------
+        # For P(Y| X1, X2) = N(0.2x1 + 0.3x2 + 1.0; 1)
```

Comment:
There's a typo here. The variable here is `Y` but the variable in the example below is `X3`